manifests/plugin/dns_rfc2136.pp

```diff
@@ -0,0 +1,59 @@
+# == Class: letsencrypt::plugin::dns_rfc2136
+#
+#   This class installs and configures the Let's Encrypt dns-rfc2136 plugin.
+#   https://certbot-dns-rfc2136.readthedocs.io
+#
+# === Parameters:
+#
+# [*server*]
+#   Target DNS server.
+# [*key_name*]
+#   TSIG key name.
+# [*key_secret*]
+#   TSIG key secret.
+# [*key_algorithm*]
+#   TSIG key algorithm.
+# [*port*]
+#   Target DNS port.
+# [*manage_package*]
+#   Manage the plugin package.
+# [*config_dir*]
+#   The path to the configuration directory.
+#
+class letsencrypt::plugin::dns_rfc2136 (
+  String $server,
+  String $key_name,
+  String $key_secret,
+  String $key_algorithm = $letsencrypt::params::dns_rfc2136_algorithm,
+  String $port          = $letsencrypt::params::dns_rfc2136_port,
```

Comment:
We should probably use this datatype? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp

Reply:
And `Stdlib::Host` for `$server`?

Reply:
Integer for `$propagation_seconds` probably wouldn't hurt as well.